daemon/daemon.go

```diff
@@ -559,7 +571,34 @@ func (d *daemon) CreateEvent(req *pb.CreateEventRequest, resp pb.Daemon_CreateEv
 		conf.Capacity = 10
 	}
 
-	return d.createEventFromConfig(conf)
+	loggerInstance := &GrpcLogger{resp: resp}
```

Comment:
This code was taken from another function right? Then I would suggest to remove the other function to prevent duplicate code

Reply:
Yeah, I thought to remove it as your suggestion however there are some usage of createEvent function internally, it is used on other points for this reason I did not remove it. I tried to express usage of it on screenshot below. Since the functions which starts with Capital and small letters differ in go,  I prefer to keep it to prevent further errors. Do you think that is it ok ?
<img width="822" alt="Screenshot 2019-08-12 at 13 19 28" src="https://user-images.githubusercontent.com/13614433/62861624-5ed44800-bd04-11e9-8370-39abb0f42142.png">

Reply:
This part of the code creates a lot of confusion, refactoring process might be useful especially for this piece of code.